libs/client/ui/pages/src/lib/profile/profile.tsx

```diff
@@ -439,54 +437,54 @@ export const Profile: React.FC<ProfileProps> = inject(userStore.storeKey)(
                 </IonItem>
               </ItemGroup>
             </div>
-          </IonGrid>
-          {!isGuest && (
-            <>
-              <IonRow className="profile__content__danger-row">
-                <IonCol className="delete">
-                  <IonButton
-                    onClick={() => handleDelete()}
-                    expand="block"
-                    mode="ios"
-                    color="danger"
-                  >
-                    Konto löschen
-                  </IonButton>
-                </IonCol>
-                <IonCol className="logout">
-                  <IonButton
-                    onClick={() =>
-                      handleLogout('/login', userStore?.user?.role)
-                    }
-                    expand="block"
-                    mode="ios"
-                  >
-                    {isLoggingOut ? (
-                      <IonLabel>
-                        <IonSpinner name="crescent" />
-                      </IonLabel>
-                    ) : (
-                      'Abmelden'
-                    )}
-                  </IonButton>
-                </IonCol>
-              </IonRow>
-              <IonFooter>
-                <IonRow className="ion-justify-content-center account-created">
-                  <IonText color="medium">
-                    Konto erstellt am{' '}
-                    {new Date(
-                      userStore?.user?.registratedAt as any
-                    ).toLocaleDateString('de-DE', {
-                      day: 'numeric',
-                      month: 'long',
-                      year: 'numeric',
-                    })}
-                  </IonText>
+            {!isGuest && (
```

Comment:
woas ned genau was du do geändert hosch aba des modal isch o numma vertical zentriert:
![image](https://user-images.githubusercontent.com/54234287/201535680-b9fe9d78-9622-4e70-926e-9f0446c3596d.png)

Reply:
najo, i hob halt die Account löschen und Abmelde buttons ins Grid dazua to so dass se glich breit sind wie da Rest - davor waren se breiter. Und du hosch anscheinend des lock-modal so gestyled das es halt immer in da Mitte vom Grid isch - ergo davor war halt des Grid grad zufällig ungefär a Seite groß
aber danke, isch ma net ufgfalla - i werds umbaua